apps/files_sharing/lib/Controller/Share20OcsController.php

```diff
@@ -117,6 +120,7 @@ public function __construct(
 		$this->notificationPublisher = $notificationPublisher;
 		$this->eventDispatcher = $eventDispatcher;
 		$this->sharingBlacklist = $sharingBlacklist;
+		$this->sharingAllowlist= $sharingAllowlist;
```

Comment:
```suggestion
		$this->sharingAllowlist = $sharingAllowlist;
```

I  thought that code-style would fail here.

Reply:
Doesn't seem so :/ Is there a way to make it fail here? I keep seeing these from time to time.